Doc/c-api/call.rst

```diff
@@ -164,7 +164,7 @@ Vectorcall Support API
    It does not check the :c:macro:`Py_TPFLAGS_HAVE_VECTORCALL` flag
    and it does not fall back to ``tp_call``.
 
-   .. versionadded:: 3.8
+   .. versionadded:: 3.9
```

Comment:
In Python 3.8.0, PyVectorcall_Call() was already defined as a regular function: https://github.com/python/cpython/blob/v3.8.0/Include/cpython/abstract.h#L141